bids-validator/validators/tsv/__tests__/validateTsvColumns.spec.js

```diff
@@ -55,4 +55,18 @@ describe('validateTsvColumns', () => {
     const issues = validateTsvColumns(tsvs, {}, [])
     assert.lengthOf(issues, 0)
   })
+  it('should generate error with empty columns', () => {
+    const tsvs = [
+      {
+        contents: '\t',
+        file: file,
+      },
+      {
+        contents: '\t\t',
+        file: file,
+      },
+    ]
+    const issues = validateTsvColumns(tsvs, {})
```

Comment:
Extra parameter added in ce5a154ff8d1365bf9d427abf5d31a0351544ecf.

```suggestion
    const issues = validateTsvColumns(tsvs, {}, {})
```

We should update the docs as well:

https://github.com/bids-standard/bids-validator/blob/d048d087a1ca4e7cac0310da275cd4a45e1c425a/bids-validator/validators/tsv/validateTsvColumns.js#L48-L53